src/main/kotlin/net/irisshaders/lilybot/extensions/events/LogUploading.kt

```diff
@@ -189,10 +195,25 @@ class LogUploading : Extension() {
 		}
 	}
 
+	/**
+	 * Setting these to null is necessary in case a value is missing, which would cause an error.
+	 *
+	 * @param success Whether the log upload was a success or not
+	 * @param id The ID of the log uploaded
+	 * @param error Any errors that were returned by the upload
+	 *
+	 * @since 3.1.0
+	 */
 	@kotlinx.serialization.Serializable
```

Comment:
Why is this not just `@Serializeable` with a specific import?

Reply:
Cape had import issues, not relevant